### DISCUSSION TITLE
sending FormData as "application/json" doesn't work as intended

### DISCUSSION THREAD

**Discussion by asaah18** [axios/axios#4977] [Q&A] [answered]
I am trying to send the FormData that I got from an html **form** in a _post request_ with _"application/json"_ content type according to the instruction [in this section of the readme](https://github.com/axios/axios#-html-form-posting-browser). But the code didn't work as intended.

### expected result

```json
{
  "foo": "1",
  "deep": {
    "prop": {
      "spaced": "3"
    }
  },
  "baz": [
    "4",
    "5"
  ],
  "user": {
    "age": "value2"
  }
}
```

### actual result

```json
{
  "0": {
    "_value": "1"
  },
  "1": {
    "_value": "2"
  },
  "2": {
    "_value": "3"
  },
  "3": {
    "_value": "4"
  },
  "4": {
    "_value": "5"
  },
  "5": {
    "0": {
      "_value": "value1"
    },
    "1": {
      "_value": "value2"
    },
    "2": {
      "_value": "value3"
    }
  },
  "6": {
    "_value": "Save"
  },
  "_vei": {}
}
```

**Comment by DigitalBrainJS** [comment, marked as answer]
This will not work as this feature is not available in version 0.27.2. This documentation is for version 1.0.0-alpha.1.
